game.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link href="https://fonts.googleapis.com/css?family=Diplomata+SC|Griffy|Roboto" rel="stylesheet"/>
+    <link href="style.css" rel="stylesheet"/>
+    <title>
+    </title>
+  </head>
+  <body>
+
+<h1>Header</h1>
```

Comment:
you have some indentation issues in both files.  elements should be indented to one level more than their parent container, and to an equal level as all sibling elements within that container.